.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,7 @@ A few sentences and/or a bulleted list to describe and motivate the change:
 - [ ] Change implemented (can be split into several points),
 - [ ] update docstring (if appropriate),
 - [ ] update user guide (if appropriate),
-- [ ] add entry to `CHANGES.rst` (if appropriate),
+- [ ] add an changelog entry in the `upcoming_changes` folder (see `upcoming_changes/README.rst`),
```

Comment:
One final thing, would it be possible to add a link to the README.rst file so that the reader can just click on it?

Reply:
Yes, make sense, the link is broken at the moment but it should work once, this PR reaches `RELEASE_next_minor`: I extrapolated what would be the link based on https://github.com/ericpre/hyperspy/blob/use_towncrier_changelog/upcoming_changes/README.rst